images/grafana-mimir/config/main.tf

```diff
@@ -0,0 +1,31 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "extra_packages" {
+  description = "Additional packages to install."
+  type        = list(string)
+  default     = ["grafana-mimir"]
+}
+
+module "accts" {
+  source = "../../../tflib/accts"
+  run-as = 65532
+  uid    = 65532
+  gid    = 65532
+  name   = "nonroot"
```

Comment:
```suggestion
```

I believe these are already the defaults.

Reply:
Addressed (went ahead and removed the accounts override from config entirely). Thanks Jason!